SPOWebService/DDMS.WebService.DDMSOperations/Controllers/DDMSController.cs

```diff
@@ -14,7 +15,7 @@
 namespace DDMS.WebService.DDMSOperations.Controllers
 {
     [ExcludeFromCodeCoverage]
-    //[Authorize(Roles = "DDMS")]
+    [Authorization]
```

Comment:
@NethraKonkamurthy  This is not what I wanted.  I wanted to pass the Application name as parameter.